gcc/cp/contracts.cc

```diff
@@ -1618,15 +1618,15 @@ contract_any_active_p (tree contract)
   return false;
 }
 
-/* Do we need to mess with contracts for DECL1?  */
+/* Do we need to mess with contracts for DECL?  */
 
 static bool
-handle_contracts_p (tree decl1)
+handle_contracts_p (tree decl)
 {
```

Comment:
Sometimes we clean up single-param functions to rename a param from decl1 to decl..